content/en/docs/home/_index.md

```diff
@@ -15,7 +15,7 @@ menu:
     title: "Documentation"
     weight: 20
     post: >
-      <p>Learn how to use Kubernetes with conceptual, tutorial, and reference documentation. You can even <a href="/editdocs/" data-auto-burger-exclude>help contribute to the docs</a>!</p>
+      <p>Learn how to use Kubernetes with conceptual, tutorial, and reference documentation. You can even <a href="/editdocs/" data-auto-burger-exclude data-proofer-ignore>help contribute to the docs</a>!</p>
```

Comment:
@celestehorgan ,
I saw a note online about, `data-proofer-ignore`,
`Add the data-proofer-ignore attribute to any tag to ignore it from every check. The name of this attribute can be customised.`
How did you determine what needed to be ignored?

Reply:
In general, I did the following:

1) Ignore any links generated in template files: this includes menus, submenus, and other permalinks in the site's "Chrome". Excluding these is _super_ important, because it drops the total errors for ~1000s to less than 100. 
2) Ignore any other links that show up as broken on _every_ page, or on a majority of pages. If the same link shows up as broken/an error 100s of times across the documentation set, then we can deduce that – somehow – it's getting thrown in as a part of the template or page's data, somehow.

For some reason, the link in this header file renders as a broken link on every page, ergo I applied rule #2 to this particular link.

Reply:
Q: Will similar changes need to be made in some of the new theme pages?

Reply:
@kbhawkey yes.